### ISSUE TITLE
Biopython Warning

### ISSUE THREAD

**Issue by and3k** [jfjlaros/demultiplex#3]
Hi!

really great package, thank you!

I am getting a BiopythonWarning when I use demux, see example below.

```
$ demultiplex demux -m 0 -r -s 1 -e 12 barcodes.tsv Read1.fastq Read2.fastq
~/virtualenvs/demultiplex_test/lib/python3.7/site-packages/Bio/Seq.py:163: BiopythonWarning: Biopython Seq objects now use string comparison. Older versions of Biopython used object comparison. During this transition, please use hash(id(my_seq)) or my_dict[id(my_seq)] if you want the old behaviour, or use hash(str(my_seq)) or my_dict[str(my_seq)] for the new string hashing behaviour.
  "the new string hashing behaviour.", BiopythonWarning)
```

So far, demuxing seems to work, but would be great if this could be fixed, as we want to use it in one of our pipelines :)

Thanks!

Best,
Bela

**Comment by jfjlaros** [comment]
Dear Bela,

Thank you for raising this issue. I have addressed this in 9e6bb7e9.

fixes: #3

**Comment by and3k** [comment]
Thank you!
